chapters/equations.tex

```diff
@@ -207,7 +207,7 @@ \subsection{When-Equations}\label{when-equations}
 \begin{lstlisting}[language=modelica]
 equation
   when x > 2 then
-    y3 = 2*x +y1+y2; // Order of y1 and y3 equations does not matter
+    y3 = 2 * x + y1 + y2; // Order of y1 and y3 equations does not matter
```

Comment:
```suggestion
    y3 = 2*x +y1+y2; // Order of y1 and y3 equations does not matter
```
No need to change this line.

Reply:
Well, the spacing was truly bad, it is tangentially related to what is being addressed in this PR, and fixing it here helps keeping down the number of open PRs.

Reply:
I don't agree that the spacing is truly bad.

Reply:
And I also believe that having different formatting in the specification is a benefit in itself (within reason), as we are standardizing a language - not a specific formatting.

Reply:
That has some merit, but not a reason to even have inconsistency within a single equation.  This would be better, and at the same time break with the "space around binary operator" style:
```suggestion
    y3 = 2*x + y1 + y2; // Order of y1 and y3 equations does not matter
```